lib/csv_helper.rb

```diff
@@ -1,15 +1,6 @@
 module CSVHelper
 
-  CSV = case RUBY_VERSION
-        when "1.8.7"
-          require "faster_csv" unless defined?(FasterCSV)
-
-          FasterCSV
-        else
-          require "csv" unless defined?(CSV)
-
-          CSV
-    end
+  require "csv"
```

Comment:
🎉  I fully support finally getting rid of this. Do we need the explicit `require "csv"` here?